Lib/pstats.py

```diff
@@ -416,7 +416,7 @@ def print_call_heading(self, name_size, column_title):
                 subheader = isinstance(value, tuple)
                 break
         if subheader:
-            print(" "*name_size + "    ncalls  tottime  cumtime", file=self.stream)
+            print(" "*name_size + "       ncalls     tottime     cumtime", file=self.stream)
```

Comment:
Why don't replace it for `\t\t`?

Reply:
Because I had completely forgot about using tabs. But, indeed, here tabs should excel their best.

Reply:
Actually, I just tried that out, and they aligned very off.